### ISSUE TITLE
Problème affichage fiche zone sensible multipolygone

### ISSUE THREAD

**Issue by thomaspnrhj** [GeotrekCE/Geotrek-admin#3589]
Quand on zoome/dézoome dans la fiche d'une zone sensible, et si la zone est multipolygone, on a des bugs d'affichage avec le fond (testé sur notre instance + Biodiv'Sport)
![image](https://user-images.githubusercontent.com/19687397/236409751-1f742f80-efb6-4893-ac96-d7f0ddec8f9a.png)
![image](https://user-images.githubusercontent.com/19687397/236409849-848518d7-65dd-4f27-9303-bd2a79e4dbe3.png)

**Comment by camillemonchicourt** [comment]
Exact, je reproduis bien sur https://geotrekdemo.ecrins-parcnational.fr/sensitivearea/433/ (admin / admin)
